### ISSUE TITLE
Importing records into an existing project fails

### ISSUE THREAD

**Issue by simonepstein** [code-kern-ai/refinery-gateway#63]
Steps to reproduce:
* Create a sample project, e.g. "Clickbait - initial"
* Label a record
* Download the records as export.json
* Upload the records
* Observe record import fails

The following is seen in the logs for the refinery-gateway docker container:

```
  File "/app/./api/transfer.py", line 53, in post
    init_file_import(task, project_id, is_global_update)
  File "/app/./api/transfer.py", line 122, in init_file_import
    upload_manager.import_records(project_id, task)
  File "/app/./controller/transfer/manager.py", line 47, in import_records
    import_file(project_id, task)
  File "/app/./controller/transfer/record_transfer_manager.py", line 86, in import_file
    ) = split_record_data_and_label_data(chunk)
  File "/app/./controller/transfer/record_transfer_manager.py", line 241, in split_record_data_and_label_data
    item.strip() == ""
AttributeError: 'list' object has no attribute 'strip'

```

My use case here is that I wanted to re-import the labelling I have done in a similar project into a new project with a schema with an additional field.  It failed in a similar way (float has no attribute 'strip')  I am unsure if this use case is supported, but it seems like roundtripping the records should work if a record has a primary key defined.

**Comment by JWittmeyer** [comment]
Hi @simonepstein 
the record export isn't meant to be imported back at the moment. This is related to diverging formats. I've added it to enhancements. What is the specific usecase you are trying to achieve so I can maybe give you a workaround for the meantime?

**Comment by simonepstein** [comment]
Hi there, the use case is that I have imported some data, done some labelling and build some heuristics to provide input into the weak supervision.  Then I have realised that there is some additional data that I would like to add to the existing records (a new field in the schema) as I have thought of an additional heuristic that I think will improve the weak supervision results.  The existing labelling is correct, so I want to avoid having to rekey it possible.

It looks like I can just import the CSV file with the extra columns into my existing project and it works just fine; the labelling is preserved and the new columns are added to the schema.  I think I got thrown off by the pk issue and then went down the export/import rabbit hole instead.  I will close this ticket as you have the export/import logged under enhancements.
